.NET/Microsoft.Recognizers.Text.NumberWithUnit/Extractors/NumberWithUnitExtractor.cs

```diff
@@ -79,6 +79,25 @@ public List<ExtractResult> Extract(string source)
             {
                 var numbers = this.config.UnitNumExtractor.Extract(source).OrderBy(o => o.Start);
 
+                if (numbers.Count() > 0 && CheckExtractorType(Constants.SYS_UNIT_CURRENCY))
+                {
+
+                    foreach (var number in numbers)

```

Comment:
Can you add a code comment here explaining what this is doing?
Also, can you check if there's measurable performance impact?
Thanks.

Reply:
ok